Update policy generator parameter table (#5995)
open-cluster-management/rhacm-docs: governance/policy_generator.adoc

| `policyDefaults.namespaceSelector`
| Required for namespaced objects that do not have a namespace specified

> | Determines namespaces in the managed cluster that the object is applied to. The `include` and `exclude` parameters accept file path expressions to include and exclude namespaces by name. The `matchExpressions` and `matchLabels` parameters specify namespaces to include by label. Read the link:https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/[_Kubernetes labels and selectors_] documentation. The resulting list is compiled by using the intersection of results from all parameters.

Author: I wasn't sure whether a link should be here or not (since it's in the "Additional resources"), so I can revert this if needed.

Reviewer: Since it is in the Additional resources section and we are decreasing inline links, the change here should be reverted

| `policyDefaults.evaluationInterval`
| Optional
| Use the parameters `compliant` and `noncompliant` to specify the frequency for a policy to be evaluated when in a particular compliance state. When managed clusters have low CPU resources, the evaluation interval can be increased to reduce CPU usage on the Kubernetes API. These are in the format of durations. For example, `"1h25m3s"` represents 1 hour, 25 minutes, and 3 seconds. These can also be set to "never" to avoid evaluating the policy after it has become a particular compliance state.

| `policyDefaults.pruneObjectBehavior`
| Optional

> |  Determines whether objects created or monitored by the policy should be deleted when the policy is deleted. Pruning only takes place if the remediation action of the policy has been set to "enforce". Example values are "DeleteIfCreated", "DeleteAll", or "None". This defaults to unset, which is equivalent to "None".

Reviewer: ```suggestion
| Determines whether objects created or monitored by the policy should be deleted when the policy is deleted. Pruning only takes place if the remediation action of the policy has been set to `enforce`. Example values are `DeleteIfCreated`, `DeleteAll`, or `None`. The default value is `None`.
```

| Optional
| Set `informGatekeeperPolicies` to false to use Gatekeeper manifests directly without defining it in a configuration policy. When the policy references a violated Gatekeeper policy manifest, an additional configuration policy is generated in order to receive policy violations in {product-title-short}. The default value is `true`.

> | `policyDefaults.informKyvernoPolicies`

Reviewer: Should this parameter be removed? Kyverno policies was removed: https://issues.redhat.com/browse/ACM-7893
